build.gradle

```diff
@@ -825,6 +889,13 @@ uploadArchives {
                 authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
             }
 
+            addFilter('gatk') {artifact, file ->
+                artifact.name == 'gatk'
+            }
+            addFilter('gatk-test-utils') {artifact, file ->
+                artifact.name == 'gatk-test-utils'
+            }
+
```

Comment:
Can we factor this out into a named (def =) closure so it can be shared ?

Reply:
Yes, done, although it doesn't save that much typing.